src/operations/mutations/ProjectMutations.js

```diff
@@ -57,6 +57,7 @@ export const UPDATE_PROJECT = gql`
         $github_url: String,
         $expected_budget_timeframe: String,
         $toggl_url: String
+        $is_active: Boolean
```

Comment:
Maybe set it to true by default in the resolver?

Reply:
What would be the reason to set it as true by default?